graylog2-web-interface/src/views/components/searchbar/QueryInput.tsx

```diff
@@ -41,15 +47,17 @@ type Props = {
   onChange: (query: string) => Promise<string>,
   onExecute: (query: string) => void,
   placeholder?: string,
+  streams?: Array<string>,
+  timeRange?: TimeRange | NoTimeRangeOverride,
   theme: DefaultTheme,
   value: string,
   warning?: QueryValidationState,
 };
 
-const defaultCompleterFactory = (completers) => new SearchBarAutoCompletions(completers);
+const defaultCompleterFactory = (completers, timeRange, streams) => new SearchBarAutoCompletions(completers, timeRange, streams);
```

Comment:
Since the search query is only reflecting the selected time range and streams after submitting the form, we can not use a store inside `FieldValueCompletion` to get the information.

Reply:
While you are at it, can you type the function arguments?